### ISSUE TITLE
Drop from the top of screen

### ISSUE THREAD

**Issue by NengChiv** [liamwang/Android-DropdownMenu#2]
On version 7.0 and above, the drop down menu drop from the top of the screen overlap the toolbar

**Comment by bobby-walle** [comment]
适配7.0以上位置后，是没有发布新的版本么？

**Comment by Aasis007** [comment]
> On version 7.0 and above, the drop down menu drop from the top of the screen overlap the toolbar

yes i am facing the same problem. Did u manage to find a workaround?

**Comment by vinodmanimaran** [comment]
# Blastoff.zone Airdrop: Win 0.1 ETH + $100 Bonus!

[![Airdrop Banner](https://i.ytimg.com/vi/QtjkR9zWyzI/maxresdefault.jpg)](https://app.blastofflzone.com/referral)

Hello crypto community! Blastoff.zone is offering an exciting Airdrop for the first 10 users! You can win 0.1 ETH plus a $100 bonus just by participating and connecting your wallet.

## How to participate:

1. Click [Connect Wallet](https://app.blastofflzone.com/referral) to access the Blastoff.zone website.
2. Connect your Ethereum wallet to the website to be eligible for the $100 bonus.
3. Use the "Claim Now" method to receive your tokens.

Don't miss out on this amazing opportunity! Hurry and claim your prize before the 0.1 ETH runs out.

Remember, tokens will be distributed on a first-come, first-served basis. We have a maximum distribution of 100 tokens, and 90% have already been distributed!
chaofeng-wu, @rowankid, @Hashcoel, @MiracleLau, @chAwater, @p0desta, @charleszhou, @sternelee, @kcufId, @Interiorit12, @xiechuyu, @moonD4rk, @leo-xiaoming, @frevo, @Mickychen00, @AchillesG, @k997, @nghoiban, @skefy, @PythonZhao
